nohup: ignoring input
2023-04-17 00:38:13.560655: I tensorflow/core/util/port.cc:110] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2023-04-17 00:38:13.631589: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-04-17 00:38:14.606497: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Experiment dir : logs/eval-EXP-20230417-003815
04/17 12:38:15 AM device = cuda
04/17 12:38:15 AM args = Namespace(debug_mode=0, dataset='criteo', data_path='/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets', bsize=2, lr=0.0001, l2=3e-08, report_freq=1000, gpu=0, epochs=10, load='logs/search-EXP-20210326-102728', save='logs/eval-EXP-20230417-003815', alpha_mode=1, model='DeepFM', orig_embedding_dim=20, comb_embedding_dim=10, X=20, Y=20, comb_field=325)
04/17 12:38:19 AM param size = 0.128132 GB
04/17 12:38:19 AM model = DeepFM(
  (cate_embeddings_table): Embedding(514798, 20)
  (fc_layers): ModuleList(
    (0): Linear(in_features=780, out_features=700, bias=True)
    (1-4): 4 x Linear(in_features=700, out_features=700, bias=True)
  )
  (norm_layers): ModuleList(
    (0-4): 5 x LayerNorm((700,), eps=1e-05, elementwise_affine=True)
  )
  (output_layer): Linear(in_features=700, out_features=1, bias=True)
)
====== epoch: 0/10
04/17 12:38:19 AM epoch 0 lr 1.000000e-04
2023-04-17 00:38:19.233868: I tensorflow/compiler/xla/service/service.cc:169] XLA service 0xba2f7d0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2023-04-17 00:38:19.233991: I tensorflow/compiler/xla/service/service.cc:177]   StreamExecutor device (0): Host, Default Version
2023-04-17 00:38:19.234989: W tensorflow/core/common_runtime/gpu/gpu_bfc_allocator.cc:47] Overriding orig_value setting because the TF_FORCE_GPU_ALLOW_GROWTH environment variable is set. Original config value was 0.
2023-04-17 00:38:19.235101: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1635] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 35228 MB memory:  -> device: 0, name: NVIDIA A100-SXM4-40GB, pci bus id: 0000:00:04.0, compute capability: 8.0
2023-04-17 00:38:19.237085: I tensorflow/compiler/xla/service/service.cc:169] XLA service 0xbaac6a0 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2023-04-17 00:38:19.237196: I tensorflow/compiler/xla/service/service.cc:177]   StreamExecutor device (0): NVIDIA A100-SXM4-40GB, Compute Capability 8.0
2023-04-17 00:38:19.306698: I tensorflow/core/common_runtime/executor.cc:1197] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [2]
	 [[{{node Placeholder/_0}}]]
2023-04-17 00:38:19.307063: I tensorflow/core/common_runtime/executor.cc:1197] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [2]
	 [[{{node Placeholder/_0}}]]
==== train: dataset_folder:/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39
get_data:  /content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39/train*.tfrecord
files: ['/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39/train_part_0.tfrecord', '/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39/train_part_1.tfrecord']
04/17 12:38:34 AM train 1000 4.812453e-01
04/17 12:38:34 AM Time 15.000401
04/17 12:38:41 AM train_loss 0.477066
2023-04-17 00:38:41.450721: I tensorflow/core/common_runtime/executor.cc:1197] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [1]
	 [[{{node Placeholder/_0}}]]
2023-04-17 00:38:41.451119: I tensorflow/core/common_runtime/executor.cc:1197] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [1]
	 [[{{node Placeholder/_0}}]]
==== infer: dataset_folder:/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39
get_data:  /content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39/test*.tfrecord
files: ['/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39/test_part_0.tfrecord']
04/17 12:38:44 AM test_auc 0.782682
04/17 12:38:44 AM test_log_loss 0.469164
04/17 12:38:44 AM Epoch time: 25.639464 s
04/17 12:38:44 AM Train time: 22.208378 s
04/17 12:38:44 AM Test time: 3.013592 s
====== epoch: 1/10
04/17 12:38:44 AM epoch 1 lr 1.000000e-04
2023-04-17 00:38:44.881753: I tensorflow/core/common_runtime/executor.cc:1197] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [2]
	 [[{{node Placeholder/_0}}]]
2023-04-17 00:38:44.882122: I tensorflow/core/common_runtime/executor.cc:1197] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [2]
	 [[{{node Placeholder/_0}}]]
==== train: dataset_folder:/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39
get_data:  /content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39/train*.tfrecord
files: ['/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39/train_part_0.tfrecord', '/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39/train_part_1.tfrecord']
04/17 12:38:58 AM train 1000 4.554990e-01
04/17 12:38:58 AM Time 13.589755
04/17 12:39:05 AM train_loss 0.454081
2023-04-17 00:39:05.416503: I tensorflow/core/common_runtime/executor.cc:1197] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [1]
	 [[{{node Placeholder/_0}}]]
2023-04-17 00:39:05.416931: I tensorflow/core/common_runtime/executor.cc:1197] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [1]
	 [[{{node Placeholder/_0}}]]
==== infer: dataset_folder:/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39
get_data:  /content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39/test*.tfrecord
files: ['/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39/test_part_0.tfrecord']
04/17 12:39:08 AM test_auc 0.782871
04/17 12:39:08 AM test_log_loss 0.470902
04/17 12:39:08 AM Epoch time: 24.038956 s
04/17 12:39:08 AM Train time: 20.534403 s
04/17 12:39:08 AM Test time: 3.019984 s
====== epoch: 2/10
04/17 12:39:08 AM epoch 2 lr 1.000000e-04
2023-04-17 00:39:08.924207: I tensorflow/core/common_runtime/executor.cc:1197] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [2]
	 [[{{node Placeholder/_0}}]]
2023-04-17 00:39:08.924788: I tensorflow/core/common_runtime/executor.cc:1197] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [2]
	 [[{{node Placeholder/_0}}]]
==== train: dataset_folder:/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39
get_data:  /content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39/train*.tfrecord
files: ['/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39/train_part_0.tfrecord', '/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39/train_part_1.tfrecord']
04/17 12:39:25 AM train 1000 4.355425e-01
04/17 12:39:25 AM Time 16.808374
04/17 12:39:30 AM train_loss 0.433639
2023-04-17 00:39:30.780976: I tensorflow/core/common_runtime/executor.cc:1197] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [1]
	 [[{{node Placeholder/_0}}]]
2023-04-17 00:39:30.781328: I tensorflow/core/common_runtime/executor.cc:1197] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype string and shape [1]
	 [[{{node Placeholder/_0}}]]
==== infer: dataset_folder:/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39
get_data:  /content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39/test*.tfrecord
files: ['/content/drive/MyDrive/Lehigh/Courses/DSCI 441 Statistical and Machine Learning/project/Recommendation_Systems/datasets/Criteo-new/X_20/orig_39/test_part_0.tfrecord']
04/17 12:39:32 AM test_auc 0.773041
04/17 12:39:32 AM test_log_loss 0.491428
04/17 12:39:32 AM Early stopped!
04/17 12:39:32 AM best_test_auc 0.782871
04/17 12:39:32 AM best_log_loss 0.470902
